src/libcore/panicking.rs

```diff
@@ -34,25 +34,25 @@ use fmt;
 
 #[cold] #[inline(never)] // this is the slow path, always
 #[lang="panic"]
-pub fn panic(expr_file_line: &(&'static str, &'static str, uint)) -> ! {
+pub fn panic(expr_file_line: &(&'static str, &'static str, usize)) -> ! {
```

Comment:
This should be either `u32` or `u64`, you could make an argument for either -- I'd suggest `u64` based on the maximum file length we report and the lack of performance issues here.